Add Newbie Op Guide (#15)
smrealms/wiki-data: tutorials/newbie-op.md

# Operations for Newbies

### Am I ready to op?
Some common instructions are: 
 - plot to sector X
 - go repair at the nearest location
 - follow the flag
 - drop mines in X sector
 - follow the path clear of mines
 - add/remove shields/drones to/from a planet
 - clear mines to X sector

If you’re comfortable with those types of tasks you will do well.  If not I would suggest practicing on [the beta site](https://beta.smrealms.de).  Turns are limitless there.

### Golden Rules
Following these three rules will lead to the correct action in nearly all situations:  
1: don’t fire at ports or planets unless you are the trigger  
2: stay with the group  
3: follow instructions of op leader  (this rule will sometimes override the others)  

A bit of the why behind those rules:
 - Ports and planets are big and have a lot of firepower.  Time must be taken to assess/repair damaged ships between shots.  To keep order an op leader will designate a single person responsible for firing.  This person is colloquially called a trigger.

>  - Operations are a structured form of group combat.  The key word there is group.  If you find your way in to combat all by yourself you’re likely do die.  The safest place to be is with your alliance mates.  Op leaders will often set a flag on their sector that indicates position.  Use the local map to watch it.

Reviewer: ```suggestion
 - Operations are a structured form of group combat.  The key word there is group.  If you find your way into combat all by yourself you’re likely to die.  The safest place to be is with your alliance mates.  Op leaders will often set a flag on their sector that indicates position.  Use the local map to watch it.
```

 - The op leader needs to have tactical control.  He or she will need to order groups or individual ships to move to positions or complete tasks vital to operational success.  A tactical blunder made as a group will usually be more successful than a cunning move executed as an individual.

### What to do when you die:
Ask the op leader if you’re returning.  If the op is nearly over you may just be done.  If it’s still going on a while you’ll need to buy a new ship, arm it, and return to the group.  

> You should make a note of the locations you need to re-arm.  Bank, ship dealer, uno/drones, any special equipment you need, and weapon shops.  You should also know ahead of time how much money it takes to get all those things.  And you should end up with 3-4 million extra so you can keep repairing.  This planning is an underrated time saver.

Reviewer: ```suggestion
You should make a note of the locations you need to re-arm.  Bank, ship dealer, uno/drones, any special equipment you need, and weapon shops.  You should also know ahead of time how much money it takes to get all those things.  And you should end up with 3-4 million extra so you can keep repairing.  This planning is an underrated time saver. It is often helpful to make a note of important locations in your "Stored Destinations" on the "Plot A Course" page.
```
